index.html

```diff
@@ -29,57 +31,56 @@ <h1 class="w3-display-topmiddle w3-container">Trick</h1>
     <!-- Host will switch to page where HOST can set up game with OPTIONS -->
     <!-- Guest player will be switching to a new page to enter the CODE from HOST  -->
     <main>
-        <section id='landing-page' class='page hidden'>
+        <section id='landing-page' class='page w3-display-container w3-center'>
             <h2>Welcome to Trick Trivia!</h2>
             <span id="player-name">
-                <label for="player-name-input">Player Name: </label>
-                <input id="player-name-input" type="text">
+                <label for="player-name-input">Player: </label>
+                <input id="player-name-input" type="text" placeholder=" Sign In">
             </span>
 
-            <div id="enter-game-options">
+            <div id="enter-game-options" class="w3-margin-top">
                 <button id='host-game-button'>Host Game</button>
                 <button id='join-game-button'>Join Game</button>
+                <button id="show-rule-hover" class="w3-margin-top" onclick="showRuleButton()">Rules</button>
             </div>
 
             <!-- Rule page will be hidden. Instead it only show a button which will be on the top RIGHT/LEFT side of the Welcome page -->
             <!-- Player will be CLICK on the the RULE Button to view the rules -->
             <!-- Once Player click button, it will display pop-up a BLOCK page with the RULE DISPLAY -->
-            <div class="w3-container">
-                <button id="show-rule-hover" class="w3-button w3-medium" onclick="showRuleButton()">Rules</button>
-                <div id="game-rule-text" class="w3-border w3-round-large">
+            <div class="game-rule-card">
+                <div id="game-rule-text" class="w3-left-align">
                     <h4>GOAL</h4>
                     <p>
-                        The goal of this game is to trick your friends and get the most points to
-                        win. There are two stages to this game; setup and guessing.
+                        Try and trick your friends with the best wrong answers to earn the most points. 
                     </p>
                     <h4>Stage-1</h4>
                     <p>
-                        In the setup stage, you will be shown a question and you will be able
-                        to provide a believable answer that is <strong>NOT</strong> correct.
-                        You will only have 30 seconds to come up with the answer and submit it.
+                        In <strong>Trick Trivia</strong>, players will be shown a question and then
+                        provide the best <strong>wrong</strong> answer.
```

Comment:
Could you change this to "...then you will need to provide...". The way it's written right now, it sounds like the game will be providing the wrong answer to the player instead of the other way around.

Reply:
This was from the original rules description. I have that changed on my end.

Reply:
Never mind. Updated.